pkg/vsphere/vm/vm.go

```diff
@@ -458,18 +471,18 @@ func (vm *VirtualMachine) registerVM(ctx context.Context, path, name string,
 }
 
 // FixInvalidState fix vm invalid state issue through unregister & register
-func (vm *VirtualMachine) FixInvalidState(ctx context.Context) error {
+func (vm *VirtualMachine) fixVM(ctx context.Context) error {
 	log.Debugf("Fix invalid state VM: %s", vm.Reference())
 	folders, err := vm.Session.Datacenter.Folders(ctx)
 	if err != nil {
 		log.Errorf("Unable to get vm folder: %s", err)
 		return err
 	}
 
-	properties := []string{"config.files", "summary.config", "summary.runtime", "resourcePool", "parentVApp"}
+	properties := []string{"summary.config", "summary.runtime", "resourcePool", "parentVApp"}
```

Comment:
While you're in there, looks like we only use Runtime.Host here, maybe limit that to "summary.runtime.host".